src/components/lib/SideToggleComponent/SideToggleComponent.js

```diff
@@ -0,0 +1,15 @@
+export default {
+  data () {
+    return {
+      isClicked: false
+    }
+  },
+  created: function () {
+    this.isClicked = false
+  },
+  methods: {
+    ClicedOn () {
```

Comment:
Possibly a typo - ClickedOn
Also, it is common in vue.js to start method names with a letter in  lower case, so it will be 'clickedOn'
And it is a good idea to start handler functions names with the word 'handle', so you could rename it to 'handleClick'